src/lib/y2storage/yaml_writer.rb

```diff
@@ -383,12 +383,14 @@ def yaml_encryption(encryption)
     # @param filesystem [::Storage::BlkFilesystem]
     # @return [Hash{String => Object}]
     #
+    # rubocop:disable Metrics/CyclomaticComplexity
+    #
```

Comment:
I hate to repeat myself, but I do not consider rubocop a helpful tool. It is a tool that insists in insane things, yet lets the important issues go unnoticed.

Seriously, if that function is already too complex for us, we should not even think about doing anything as complex as YaST.